content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -36,17 +36,7 @@ Each individual is classified into four mutually exclusive categories:
 
 This dataset is the geospatial mapping of all cash outlets in Tanzania in 2012. Cash outlets in this case included commercial banks, community banks, ATMs, micro-finance institutions, mobile money agents, bus stations and post offices. This data was collected by FSDT.
 
-## Instructions:
-
-1. Examine the dataset. Are there any missing observations or columns where the data do not seem valid?
-
-2. Get basic descriptive statistics for the dataset.
-
-3. Create appropriate graphs to visually represent the relationship between financial services accessed (non-mobile, mobile, both) and age, gender, marital status, land ownership and type of income.
-
-4. What can you conclude about use of financial services in Tanzania so far? Which demographic are associated with mobile money use?
-
-## Variables
+## Metadata Information
```

Comment:
Is this not a part of the "about the dataset" section? Is the heading level correct?